packages/forma-36-react-components/tools/.storybook/storybook.global.css.d.ts

```diff
@@ -0,0 +1,230 @@
+// This file is automatically generated by typings-for-css-modules.
+// Please do not change this file!
+export interface IStorybookGlobalCss {
```

Comment:
Why it's produced? I should not happen according to the configuration I see

Reply:
Good catch, was an artefact from trying something else. I've also removed the `font.css.d.ts` file